jujugui/static/gui/src/app/jujulib/bakery.js

```diff
@@ -212,11 +199,11 @@ YUI.add('juju-env-bakery', function(Y) {
     */
     discharge(macaroon, onSuccess, onFailure) {
       try {
-        macaroonlib.discharge(
-          macaroonlib.import(macaroon),
+        macaroonlib.dischargeMacaroon(
```

Comment:
Is it just a naming change or are we using another lib?
Anyway, TBH I prefer discharge over dischargeMacaroon. [shrug]